test/support/environment-variables.ts

```diff
@@ -9,6 +9,14 @@ export const ENVIRONMENT_VARIABLES = Object.freeze({
   SWAGGER_USER: valueGenerator.string(),
   SWAGGER_PASSWORD: valueGenerator.string(),
   API_KEY: valueGenerator.string(),
+
+  // These are not currently used in graph-authentication.service.test.ts
```

Comment:
(If we're using this the same as we did on TFS then) this file is for the environment variables for the API tests, not for unit tests